docs/modules/releasenotes/pages/whatsnew.adoc

```diff
@@ -7,7 +7,12 @@
 * *Java 11 and 17*: OpenNMS Horizon 32 runs on JDKs 11 and 17.
 * *PostgreSQL 10 or higher*: Horizon 32 requires any supported version of PostgreSQL from 10 up to (and including) 15.
 
-== New features and improvements
+== New features and important changes
+
+=== JDK 17
+
+Horizon 32 adds support for running on JDK 17, while retaining support for JDK 11.
+We are not (yet) taking advantage of JDK 17 specific features, but the new JDK provides a bunch of performance improvements under the covers that benefit any application running on it.
 
 === Newts: DataStax driver for Cassandra upgrade
```

Comment:
Line 26 appears to be self-referential. It mentions editing `newts-on-cassandra.conf` with a property pointing to itself.  Should that reference an `opennms.properties.d` file?

Reply:
yeah, probably... @j-white wrote that -- pretty sure that should be `opennms.properties.d`